starport/services/network/launch.go

```diff
@@ -0,0 +1,62 @@
+package network
+
+import (
+	"context"
+	"fmt"
+
+	launchtypes "github.com/tendermint/spn/x/launch/types"
+	"github.com/tendermint/starport/starport/pkg/date"
+	"github.com/tendermint/starport/starport/pkg/events"
+	"github.com/tendermint/starport/starport/services/network/networkchain"
+)
+
+// LaunchParams fetches the chain launch module params from SPN
+func (n Network) LaunchParams(ctx context.Context) (launchtypes.Params, error) {
+	res, err := launchtypes.NewQueryClient(n.cosmos.Context).Params(ctx, &launchtypes.QueryParamsRequest{})
+	if err != nil {
+		return launchtypes.Params{}, err
+	}
+	return res.GetParams(), nil
+}
+
+// TriggerLaunch launches a chain as a coordinator
+func (n Network) TriggerLaunch(ctx context.Context, launchID, remainingTime uint64) error {
+	n.ev.Send(events.New(events.StatusOngoing, fmt.Sprintf("Launching chain %d", launchID)))
+
+	address := n.account.Address(networkchain.SPN)
+	params, err := n.LaunchParams(ctx)
+	if err != nil {
+		return err
+	}
+
+	switch {
+	case remainingTime == 0:
+		// if the user does not specify the remaining time, use the minimal one
+		remainingTime = params.MinLaunchTime
+	case remainingTime < params.MinLaunchTime:
+		return fmt.Errorf("remaining time %s lower than minimum %s",
+			date.Now(remainingTime),
```

Comment:
I think date pkg is a bit too much. Can't we just use `time.Now().Add(remaningTime)` here directly and rm the date pkg? 

If you still think data pkg is necessary, can we change date.Now with something more explatory like `xtime.NowAfter(time.Duration) time.Time`, we can also change the other func as `xtime.FormatAsUnix()?

Reply:
It's easier if we need to change the date format. We only need to change in one place. I change the package and method names.